docs/products/clickhouse/howto/connect-with-go.rst

```diff
@@ -144,5 +144,5 @@ Replace the placeholders in the code with meaningful information on your service
 
 .. seealso::
 
-    * For instructions on how to configure connection settings, see `Connection settings <https://clickhouse.com/docs/en/integrations/go/clickhouse-go/clickhouse-api/#connection-settings>`_.
+    * For instructions on how to configure connection settings, see `Connection settings <https://clickhouse.com/docs/en/integrations/go#connection-details>`_.
```

Comment:
Please replace the linked doc' title as follows:

"Connection settings" => "Connection Details"